.github/dependabot.yml

```diff
@@ -1,9 +1,18 @@
 version: 2
 updates:
+
+  # New `neptune` package
   - package-ecosystem: pip
     directory: "/"
     schedule:
       interval: "weekly"
+
+  # Old `neptune-client` package
+  - package-ecosystem: pip
+    directory: ".github/"
```

Comment:
Why is the directory `.github` here and in the new neptune it's `/`?

Reply:
That's where we're storing the package definition of `neptune-client` :D. Take a look at release workflow :wink:

Reply:
Currently Dependabot is looking for only `neptune` package (probably) as only our `pyproject.toml` is detected from the root directory. We would like to take into account `neptune-client` as well that was stored in `.github` directory.